Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members         = ["drivers/stspin220"]
+members         = ["drivers/drv8825", "drivers/stspin220"]
 default-members = [".", "drivers/stspin220"]
```

Comment:
Found another small problem: The new driver needs to be added to `default-members`, otherwise `cargo build` and `cargo test` on the top-level won't cover it.